docker/Dockerfile.nvidia

```diff
@@ -40,25 +43,17 @@ RUN apt-get update -y && \
         libblas-dev \
         libibverbs-dev libmlx4-1 libmlx5-1 ibutils \
         texlive-latex-extra texlive-fonts-recommended dvipng cm-super \
-        ffmpeg && \
-    wget -q -P /app/ \
-         https://developer.download.nvidia.com/hpc-sdk/21.9/nvhpc-21-9_21.9_amd64.deb \
-         https://developer.download.nvidia.com/hpc-sdk/21.9/nvhpc-21-9-cuda-multi_21.9_amd64.deb \
-         https://developer.download.nvidia.com/hpc-sdk/21.9/nvhpc-2021_21.9_amd64.deb && \
+        ffmpeg \
+        nvhpc-22-2 nvhpc-22-2-cuda-multi && \
```

Comment:
Great, I used it this morning and thought that I was forgetting commands. Nice simplification in installation.